server/i18n/en.json

```diff
@@ -335,6 +335,10 @@
     "id": "api.channel.patch_channel_moderations.license.error",
     "translation": "Your license does not support channel moderation"
   },
+  {
+    "id": "api.channel.patch_update_channel.cant_update_direct_or_group_messages.app_error",
+    "translation": "Unable to update the name, display_name, and purpose of a direct or group messages."
```

Comment:
"Unable" is not appropriate here. It means something went wrong in trying to update the field. It should rather be "You are not allowed to update the ..."

Reply:
I updated the message. I'll update the message if the `purpose` field becomes updatable.